hat/assets/js/apps/Iaso/domains/instances/details.tsx

```diff
@@ -97,7 +97,7 @@ const InstanceDetails: FunctionComponent<Props> = props => {
     // not showing history link in submission detail if there is only one version/log
     // in the future. add this info directly in the instance api to not make another call;
     const { data: instanceLogsDetails } = useGetInstanceLogs(instanceId);
-    const showHistoryLink = instanceLogsDetails?.list?.length ?? 0 > 1;
+    const showHistoryLink = instanceLogsDetails?.list?.length > 1;
```

Comment:
It shocks me that it works. I would have expected `undefined > 1 ` to throw an error. but, well, Javascript.

Reply:
![Screenshot 2022-10-03 at 10 01 59](https://user-images.githubusercontent.com/12494624/193528675-1fa08dff-c2a2-4b39-a560-76f3e14d4139.png)